src/index.html

```diff
@@ -63,7 +63,7 @@
           <section class="col-50 hero__content">
             <h1>Capture your screen</h1>
             <h2>An open-source screen recorder built with web technology.</h2>
-            <a href="https://github.com/wulkano/kap/releases/download/v2.0.0/Kap-2.0.0.dmg" class="button hero__button">
+            <a href="https://github.com/wulkano/kap/releases/download/v2.1.1/Kap-2.1.1.dmg" class="button hero__button">
```

Comment:
I think we should link to http://kap-updates.now.sh/download instead so it's always the latest

Reply:
But I think this is fine for now as the new website is almost done 💅 👍